### DISCUSSION TITLE
How to set up a development environment on WSL2

### DISCUSSION THREAD

**Discussion by CastleAndCountry** [open-metadata/OpenMetadata#17873] [Q&A]
Hello, I am struggling to set up a development environment for OpenMetadata on Windows, in particular WSL2. Is there anyone who has managed to do so successfully and is kind enough to provide some advice or guidance? Thank you.

**Comment by Prajwal214** [comment]
@CastleAndCountry Can you share the Issue you are facing while setting up the environment.

**Comment by CastleAndCountry** [comment]
Hello, @Prajwal214 I managed to set it up correctly by installing the prerequisites with the right versions.
